### PR TITLE
register missing combiner analysis script

### PR DESCRIPTION
**Pull request by Alexander-Stuckey**
Does this even do what I want it to do? Kinda just winged it based off another script.

### CODE REVIEW

File: scripts/register_migging_combiner_analysis.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    return read_vds(to_path(vds)).variant_data.s.collect()
+
+
+def main(vds: str, dataset: str):
+    aapi = AnalysisApi()
+    am = Analysis(
+        type='combiner',
+        output=vds,
+        status=AnalysisStatus('completed'),
+        sequencing_group_ids=get_sg_ids(vds),
```
Comment (MattWellie) [inline scripts/register_migging_combiner_analysis.py:41]: you'll want to import `cpg_utils.hail_batch.init_batch` and call that, you won't be able to call a Hail method without activating a hail runtime, and that helper method will deal with all the billing project/ref genome stuff

Comment (Alexander-Stuckey) [reply scripts/register_migging_combiner_analysis.py:41]: I always hecking forget that

Comment (Alexander-Stuckey) [reply scripts/register_migging_combiner_analysis.py]: Alright, that's fixed now

**Review by MattWellie — APPROVED** [review]
Looks good, happy to increase the flexibility later

**Pull request merged**